src/ensembl/webpack/webpack.config.dev.js

```diff
@@ -72,7 +72,7 @@ const devConfig = {
         secure: false
       },
       '/browser': {
-        target: 'http://localhost:4000',
+        target: 'https://staging-2020.ensembl.org',
```

Comment:
this reverts the change introduced into dev [here](https://github.com/Ensembl/ensembl-client/commit/9dd304381acf06f9d3647d3824e679d30397e679)